rllib/core/learner/learner.py

```diff
@@ -883,6 +973,31 @@ def update(
                 return results
             return reduce_fn(results)
 
+    @abc.abstractmethod
```

Comment:
The new "core" in-graph/traced method. Subclasses only need to override this method, not `update()` itself.